src/evo/mnhftx.cpp

```diff
@@ -232,7 +232,7 @@ bool CMNHFManager::UndoBlock(const CBlock& block, const CBlockIndex* const pinde
     std::vector<uint8_t> excluded_signals;
     BlockValidationState state;
     if (!extractSignals(block, pindex, excluded_signals, state)) {
-        LogPrintf("%s: failed to extract signals\n", __func__);
+        LogPrintf("CMNHFManager::ProcessBlock: failed to extract signals\n");
```

Comment:
this does not reduce any log spam :) also should be `UndoBlock`, not `ProcessBlock`. or better
```suggestion
        LogPrintf("CMNHFManager::%s: failed to extract signals\n", __func__);
```